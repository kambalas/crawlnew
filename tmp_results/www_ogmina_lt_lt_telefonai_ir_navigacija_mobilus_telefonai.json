[
  {
    "name": "Mobilusis telefonas Apple iPhone 16 Pro 6.3\" Balto titano, 128 GB",
    "price": "1147.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 16 Pro 6.3\" Balto titano 256GB",
    "price": "1274.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 16 Pro Max Natural Titanium",
    "price": "1636.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 13 Pro 6.1\" Auksinis 128 GB",
    "price": "605.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 13 Pro Max 6.7\" Mėlyna 128 GB Atnaujinta (refurbished, grade A). Grade A lygio apibūdinimas: Puiki...",
    "price": "641.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 15 128GB Black Apple, 128 GB",
    "price": "793.00",
    "error": false
  },
  {
    "name": "Mobilusis telefonas Apple iPhone 16 Pro 6.3\" Natūralus titanas, 128 GB",
    "price": "1147.00",
    "error": false
  }
]