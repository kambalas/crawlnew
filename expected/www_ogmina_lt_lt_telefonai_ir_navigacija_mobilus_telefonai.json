[
  {
    "index": 0,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  },
  {
    "name": "iPhone 13 Pro 128GB Sierra Blue",
    "price": "999",
    "error": false
  },
  {
    "name": "iPhone 13 256GB Pink",
    "price": "849",
    "error": false
  },
  {
    "name": "iPhone SE 64GB (2022) Midnight",
    "price": "479",
    "error": false
  },
  {
    "index": 2,
    "error": true,
    "tags": [
      "error"
    ],
    "content": "'str' object has no attribute 'choices'"
  }
]